src/opera/test/pge/test_dswx_pge.py

```diff
@@ -294,6 +295,23 @@ def test_dswx_pge_output_validation(self):
             if os.path.exists(test_runconfig_path):
                 os.unlink(test_runconfig_path)
 
+    def test_geotiff_filename(self):
```

Comment:
This unit test needs to be patched by copying this decorator:

 @patch.object(opera.util.img_utils, "get_geotiff_metadata", get_geotiff_metadata_patch)

Reply:
completed

Reply:
Just FYI, there was one more minor change needed to make this test work within the container: the test itself was also calling the unpatched version of `get_geotiff_metadata()`, it now calls the patched version manually so the test is consistent both in and out of a container.